Documentation/Signoffs/Motor System.md

```diff
@@ -2,32 +2,71 @@
+^ There will be 20 dividers per platform, equating to 60 total platforms.  
+
+* **Given these values, each divider will weigh 0.27375 lbs (0.365 * 0.75). The total weight of the dividers will be 16.425 lbs.**  
+
+**The hollow cylinder supporting the platforms:**  
+* **Will be made of aluminum or pvc (assuming aluminum for worst-case)**  
 
-^ Source: https://www.builditsolar.com/References/Glazing/physicalpropertiesAcrylic.pdf    Tested by ASTM D790  
+* **Will be 36- to 40-inches tall (assuming 40 inch for worst-case).**  
 
-We are going to try to use 3.175mm (1/8") thick acrylic in order to reduce the total weight. Given a flexural strength of 16,500 PSI for 6mm thickness, going to 3.175 should not make the platform flex under 10 lbs.  
+* **Will be 1/16th or 1/8th thick (assuming 1/8th inch for worst-case).**  
+^ Given the density of aluminum (2.7 g/cm^3) and given the area equation for a hollow cylinder  
 
-Weight of 3-ft diameter acrylic: Assuming 1/8'" thickness, a 3 sq. ft. square acrylic sheet weighs 6.246 lbs (0.694 lbs/sq. ft.) Given that a circle with a diameter equal to the side length of a square has 78.540% of the area, a 3-ft diameter, 3.175mm thick, acrylic circle weighs 4.906 lbs.   
+M = πhmD(r^2 - (r - t)^2)  
+where:  
+  M = mass of the hollow cylinder
+  h = height  
+  r = outer radius  
+  t = thickness  
+  mD = Mean Density of cylinder material  
+The hollow aluminum cylinder will weigh **9.622 lbs**  
 
-* **Given these estimated values, and that there will be 3 platforms in the machine, the motor must output enough torque to rotate roughly 44.717 lbs. (For calculations, we will round to 45 lbs.)**
-  * This is also assuming that the acrylic circle will be solid throughout, which will not be the case for the vending machine. The inner 1'4" diameter will be hollowed out for components and sensors. Assuming that the acrylic will be solid throughout gives us torque calculations higher than what will be reality. This makes sure that if the constraints are met, they will definitely be met during the testing. 
+**The Motor:  **
+* **Given the calculations above, must rotate a total weight of 70.858 lbs. (We will assume roughly 5% tolerance for error, so for calculations we will use 75 lbs)**  
+ 
+* **The motor will receive 24VDC from the power system.**  
 
-* **The motor will receive 24VDC from the power system.**
 * **The motor will be a stepper motor.**  
 This is because a stepper motor divides a full rotation into a certain amount of equal steps. A stepper motor can do this by having groups of coils called "phases."  
 When a "phase" is energized, the stepper motor will rotate by one step. The motor will only move on to the next step if the next "phase" group is energized.  
 
 ^ Source: https://learn.adafruit.com/all-about-stepper-motors  
+
 * **The motor will be protected at the nameplate current rating (6A).**  
```

Comment:
Typically you protect a motor at 120% of the nameplate rating. Further, overcurrent protection, as stated in the referenced standard, protects the conductors (the wires), not the device.